shaders/src/light_indirect.fs

```diff
@@ -539,20 +539,27 @@ void applyRefraction(
     // which is not the case when we'll read from the screen-space buffer
     vec3 Ft = prefilteredRadiance(ray.direction, perceptualRoughness) * frameUniforms.iblLuminance;
 #else
-    // compute the point where the ray exits the medium, if needed
-    vec4 p = vec4(frameUniforms.clipFromWorldMatrix * vec4(ray.position, 1.0));
-    p.xy = uvToRenderTargetUV(p.xy * (0.5 / p.w) + 0.5);
-
-    // perceptualRoughness to LOD
-    // Empirical factor to compensate for the gaussian approximation of Dggx, chosen so
-    // cubemap and screen-space modes match at perceptualRoughness 0.125
-    // TODO: Remove this factor temporarily until we find a better solution
-    //       This overblurs many scenes and needs a more principled approach
-    // float tweakedPerceptualRoughness = perceptualRoughness * 1.74;
-    float tweakedPerceptualRoughness = perceptualRoughness;
-    float lod = max(0.0, 2.0 * log2(tweakedPerceptualRoughness) + frameUniforms.refractionLodOffset);
-
-    vec3 Ft = textureLod(light_ssr, p.xy, lod).rgb;
+    vec3 Ft;
+    if (frameUniforms.ssrDistance > 0.0f) {
```

Comment:
This test shouldn't even be there if we're not in `REFLECTION_MODE_SCREEN_SPACE` right?